charts/streams-app/templates/log4j-configmap.yaml

```diff
@@ -0,0 +1,9 @@
+{{ if .Values.log4jConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "streams-app.name" . }}-log4j-configmap
+data:
+  log4j.properties: |
+{{ .Values.log4jConfig | indent 4 }}
```

Comment:
What is the purpose of this? Can I still configure log4j.properties via values.yaml? Or just mount a file?

Reply:
We cannot just mount a file into helm. If we use helm --set-file log4jConfig=FILE_PATH than the contents of the file are in .Values.log4jConfig.

TLDR: We change the log config by mounting a log4j.properties file with --set-file

Reply:
Ok